src/concrete/ml/onnx/onnx_utils.py

```diff
@@ -443,12 +443,17 @@ def get_op_type(node):
 
 def execute_onnx_with_numpy(
     graph: onnx.GraphProto,
+    lsbs_to_remove: Optional[List[int]],
     *inputs: numpy.ndarray,
 ) -> Tuple[numpy.ndarray, ...]:
     """Execute the provided ONNX graph on the given inputs.
 
     Args:
         graph (onnx.GraphProto): The ONNX graph to execute.
+        lsbs_to_remove (Optional[List[int]]): Contains the values of the least significant bits to
+            remove during tree traversal. The first value pertains to the first comparison (either
```

Comment:
same comment as for `get_equivalent_numpy_forward_from_onnx`, we are putting something very specific to trees in a general function, I would make it more clear (same suggestions as above), if not temporary